src/modules/Security.rb

```diff
@@ -731,35 +733,30 @@ def Import(settings)
     # (For use by autoinstallation.)
     # @return [Hash] Dumped settings (later acceptable by Import ())
     def Export
-      Builtins.eval(@Settings)
+      @Settings if @Settings.is_a?(Hash)
     end
 
     # Create a textual summary and a list of unconfigured cards
     # @return summary of the current configuration
     def Summary
       settings = deep_copy(@Settings)
-      Builtins.foreach(@do_not_test) do |key|
-        settings = Builtins.remove(settings, key)
-      end
+      @do_not_test.each { |k| settings.delete k }
```

Comment:
hm.. settings is a hash, no? Plain hashes don't have the "-" method implemented, I'm not sure if I can do it.